sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -244,14 +244,14 @@ public DigitalTwinsServiceVersion getServiceVersion() {
      * {@codesnippet com.azure.digitaltwins.core.asyncClient.updateDigitalTwin#String-List}
      *
      * @param digitalTwinId The Id of the digital twin.
-     * @param digitalTwinUpdateOperations The JSON patch to apply to the specified digital twin.
+     * @param jsonPatch The JSON patch to apply to the specified digital twin.
```

Comment:
q - do we have the update operations util from azure-core available for us to use here?

Reply:
It is still in azure core's experimental package, so not yet. Hopefully they will have it out before we GA though